src/verinfast/dependencies/walkers/composer.py

```diff
@@ -0,0 +1,53 @@
+import json
+import os
+import subprocess
+from pathlib import Path
+from typing import List
+
+from verinfast.dependencies.walkers.classes import Walker, Entry
+
+
+class ComposerWalker(Walker):
+    def initialize(self, root_path: str = "./"):
+        root_path = str(Path(root_path).absolute())
+        self.install_points: List[Path] = []
+        for p in Path(root_path).rglob('**/*.*'):
+            if p.name in self.manifest_files:
+                self.install_points.append(p)
+        for p in self.install_points:
+            target_dir = Path(p).parent
+            os.chdir(target_dir)
+            try:
+                subprocess.run(
+                    "composer install --no-dev --no-progress",
+                    capture_output=True,
+                    shell=True
+                )
+            except Exception as error:
+                self.log(f'Error with composer install: {error}')
```

Comment:
log the file name, and the contents of requirements here too, also logs by default aren't visible to the user.